docs/API.md

```diff
@@ -1139,224 +1184,235 @@ type StatsCount struct {
 
-`GET trusty/wordpress-42/meta/id-revision`
-
-```
-        {
-                "Revision": 42
-        }
+```json
+{
+    "Revision": 42
+}
 ```
 
-`GET id/meta/id-name`
+#### GET id/meta/id-name
 
-The name path returns information on the name of the id. The information is exactly that contained within the id.
+The `name` path returns information on the name of the id. The information is
+exactly that contained within the id.
 
-```
-        type Name struct {
-                Name string
-        }
+```go
+type Name struct {
+        Name string
+}
 ```
 
-Example:
-
-`GET trusty/wordpress-42/meta/id-name`
+Example: `GET trusty/wordpress-42/meta/id-name`
 
-```
-        {
-                "Name": "wordpress"
-        }
+```json
+{
+    "Name": "wordpress"
+}
 ```
 
-`GET id/meta/id-user`
+#### GET id/meta/id-user
 
-==Check for accuracy.==
-The `id-owner` path returns information on the user name in the id. This information is exactly that contained within the id.
+The `id-user` path returns information on the user name in the id. This
+information is exactly that contained within the id.
 
+```go
+type User struct {
+        User string
+}
 ```
-        type User struct {
-                User string
-        }
-```
-
-Example:
 
-`GET ~bob/trusty/wordpress-42/meta/id-owner`
+Example: `GET ~bob/trusty/wordpress-42/meta/id-user`
 
-```
-        {
-                "User": "bob"
-        }
+```json
+{
+    "User": "bob"
+}
 ```
 
-`GET trusty/wordpress-42/meta/id-owner`
+Example: `GET trusty/wordpress-42/meta/id-user`
 
+```json
+{
+    "User": ""
+}
 ```
-        {
-                "User": ""
-        }
-```
-
 
-`GET id/meta/id-series`
+#### GET id/meta/id-series
 
-The `id-series` ==changed== path returns information on the series of the id. This information is exactly that contained within the id. For bundles, this will return "bundle".
+The `id-series` path returns information on the series in the id. This
+information is exactly that contained within the id. For bundles, this will
+return "bundle".
 
+```go
+type Series struct {
+        Series string
+}
 ```
-        type Series struct {
-                Series string
-        }
-```
-
-Example:
 
-`GET ~bob/trusty/wordpress-42/meta/id-series` ==changed==
+Example: `GET ~bob/trusty/wordpress-42/meta/id-series`
 
-```
-        {
-                "Series": "trusty"
-        }
+```json
+{
+    "Series": "trusty"
+}
 ```
 
 ### Resources
 
-`POST id/resources/name.stream`
+**Not yet implemented**
+
+#### POST id/resources/name.stream
 
 Posting to the resources path creates a new version of the given stream
 for the charm with the given id. The request returns the new version.
 
-```
-        type ResourcesRevision struct {
-                Revision int
-        }
+```go
+type ResourcesRevision struct {
+        Revision int
+}
 ```
 
-`GET  id/resources/name.stream[-revision]/arch/filename`
+#### GET id/resources/name.stream[-revision]/arch/filename
```

Comment:
No. Juju-core never finalised their resources spec, so this is still in nascent form.